netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -14,6 +14,165 @@
+    TraceLoggingWrite(                                                \
+        net_ebpf_ext_tracelog_provider,                               \
+        NET_EBPF_EXT_SOCK_ADDR_REDIRECT_MESSAGE,                      \
+        TraceLoggingLevel(NET_EBPF_EXT_TRACELOG_LEVEL_INFO),          \
+        TraceLoggingKeyword(NET_EBPF_EXT_TRACELOG_KEYWORD_SOCK_ADDR), \
+        TraceLoggingString(message, "Message"),                       \
+        TraceLoggingUInt64((handle), "TransportEndpointHandle"),      \
+        TraceLoggingUInt64((protocol), "Protocol"),                   \
+        TraceLoggingIPv4Address((source_ip), "source_ip"),            \
+        TraceLoggingUInt16((source_port), "source_port"),             \
+        TraceLoggingIPv4Address((destination_ip), "destination_ip"),  \
+        TraceLoggingUInt16((destination_port), "destination_port"),   \
+        TraceLoggingIPv4Address((redirected_ip), "redirected_ip"),    \
+        TraceLoggingUInt16((redirected_port), "redirected_port"),     \
+        TraceLoggingUInt64((verdict), "Verdict"));
+
+#define NET_EBPF_EXT_LOG_SOCK_ADDR_REDIRECT_CLASSIFY_IPV6(            \
+    message,                                                          \
+    handle,                                                           \
+    protocol,                                                         \
+    source_ip,                                                        \
+    source_port,                                                      \
+    destination_ip,                                                   \
+    destination_port,                                                 \
+    redirected_ip,                                                    \
+    redirected_port,                                                  \
+    verdict)                                                          \
+    TraceLoggingWrite(                                                \
+        net_ebpf_ext_tracelog_provider,                               \
+        NET_EBPF_EXT_SOCK_ADDR_REDIRECT_MESSAGE,                      \
+        TraceLoggingLevel(NET_EBPF_EXT_TRACELOG_LEVEL_INFO),          \
+        TraceLoggingKeyword(NET_EBPF_EXT_TRACELOG_KEYWORD_SOCK_ADDR), \
+        TraceLoggingString(message, "Message"),                       \
+        TraceLoggingUInt64((handle), "TransportEndpointHandle"),      \
+        TraceLoggingUInt64((protocol), "Protocol"),                   \
+        TraceLoggingIPv6Address((source_ip), "source_ip"),            \
+        TraceLoggingUInt16((source_port), "source_port"),             \
+        TraceLoggingIPv6Address((destination_ip), "destination_ip"),  \
+        TraceLoggingUInt16((destination_port), "destination_port"),   \
+        TraceLoggingIPv6Address((redirected_ip), "redirected_ip"),    \
+        TraceLoggingUInt16((redirected_port), "redirected_port"),     \
+        TraceLoggingUInt64((verdict), "Verdict"));
```

Comment:
Why the inconsistent convention in strings?  Some use PascalCase and others use lower_snake_case.

Reply:
snake_cased.